### PR TITLE
simple doc refresh, add licence & copyright info

### PR DESCRIPTION
**Pull request by rvagg**
@ded you might want to review this initial adjustment to the README. I'm going to do some more work on it to flesh out the API like Bean & Traversty but that's going to take some time. I also have @adlwalrus keen to contribute to the API docs too but we'll work on that in my repo for now.

### CODE REVIEW

**Comment by rvagg** [comment]
FYI, WIP API docs, just a skeleton for now but will be fleshed out over time and eliminate one of the biggest complaints against Ender! https://github.com/rvagg/bonzo/tree/docs-api#bonzo

**Comment by rvagg** [comment]
& @amccollum, @connor, @st-luke, @adlwalrus and anyone else who feels up for some doc work, I'd love help over there. I can give you commit access to my Bonzo repo to make collaboration on this easier if you want to help.

The aim is to make the API docs similar across some core Ender projects, so far you Bean and Traversty are pretty well documented and have a similar _style_:
- https://github.com/fat/bean#api
- https://github.com/rvagg/traversty#api

Once we have some consistency we could even start scraping the markdown with the Ender website and host documentation for Ender libs there (**any** Ender compatible lib that adhered to certain standards perhaps). e.g. you might go to http://ender.jit.su/docs/bonzo and see the API docs rendered nicely for you there. Could make it searchable too!

**Comment by luk-** [comment]
@rvagg sure, I'd be down to help. I should have some free time over the holiday or weekend. I'll jump on irc sometime soon to discuss it more.

**Comment by luk-** [comment]
Also great idea for the ender website.

**Comment by rvagg** [comment]
That's great @st-luke, thanks!

btw, I've gone ahead and given you all commit access to my fork anyway. Treat it like a wiki for now & don't aim for perfection, we can clean it up as we go along, making a start is the important bit.

Look at Bean & Traversty for inspiration on how to lay stuff out and how much detail to give, and of course you'll have to look at the Bonzo source to actually figure stuff out.

**Comment by rvagg** [comment]
It's all in the _docs-api_ branch in my fork for now btw so just mess around in there.

https://github.com/rvagg/bonzo/tree/docs-api#readme

**Comment by rvagg** [comment]
OH! and https://github.com/rvagg/morkdown if you can get it going on your system is so nice for MD work! I use `morkdown -w README.md &` and set it to the right of my Sublime Text 2 window. Then on each save it renders for me just like it does on GitHub.

**Comment by connor** [comment]
Yes! Thank you, @rvagg. Is there an IRC channel we should hop on to chat about any of this stuff? I'll definitely have time over the holiday to refresh/add some docs. This will be great.

**Comment by rvagg** [comment]
Yes, we hang on #enderjs (keep in mind that my timezone GMT+10 even though I have a client on there constantly)

**Comment by aawwawa** [comment]
Cool, cool, cool.
- I'm online sporadically throughout the day here, during which times I have IRC running. Feel free to mention my nick if you see it.
- **+1** for a fancier, even if only slightly, stylesheeted, website! I think it would be good to have that in addition to just plain B&W github READMEs.
- @rvagg Any idea if morkdown is known to run without fuss on windows? (Please don't smack me.)

**Comment by rvagg** [comment]
@adlwalrus I believe Morkdown may work on Windows. If you have Node installed then `npm install morkdown -g` and see how you go. It runs on App.js which distributes binaries via npm based on the Chromium core. The main platform they have issues with is OS X 64-bit cause Chromium is only 32-bit on OS X so far! Apparently it'll run 32-bit on 64-bit fine but not when installed via npm (or something like that). But who really cares about OS X tho eh?
I'm a bit behind on the App.js dependency so perhaps things are even better with the newer version. Will upgrade when I have time (and remember).

**Comment by connor** [comment]
I know the bonzo with updated docs lives under @rvagg's github profile, but I've stubbed out some of this stuff [here](https://github.com/rvagg/bonzo/tree/connor-API_docs), and will start actually writing the docs and whatnot tonight, and get finished up this weekend. Feel free to help!

**Comment by rvagg** [comment]
hey @connor, I already did that, and a bit more, on the _docs-api_ branch: https://github.com/rvagg/bonzo/tree/docs-api
would you be able to back up and start from the tip of that branch instead?

**Comment by connor** [comment]
...I now see you definitely mentioned that up in a comment. Lost track of that one. Sorry!

**Comment by ded** [comment]
whoa... this must have all happened during the vacation. hot damn. new docs!

File: Makefile [hunk 1]
```diff
@@ -1,2 +1,2 @@
 boosh:
-	smoosh make make/build.js
\ No newline at end of file
+	node_modules/smoosh/bin/smoosh make make/build.js
```
Comment (ded) [inline Makefile:2]: we should really really start standardizing how we build these too. not for now tho

File: README.md [hunk 2]
```diff
@@ -33,9 +53,11 @@ This now allows you to write the following code:
 $('#content a[rel~="bookmark"]').after('√').css('text-decoration', 'none');
 ```
 
-Bonzo Extension API
+<a name="extensions"></a>
```
Comment (ded) [inline README.md:56]: i'm guessing others are doing it this way. does it strip out id's?

File: README.md [hunk 3]
```diff
@@ -138,55 +161,90 @@ Added in the Ender bridge
...
-    $ npm install -d
-    $ make
+You should only edit the files in the *src/* directory. Bonzo is compiled into the *bonzo.js* and *bonzo.min.js* files contained in the root directory by the build command:
 
-Tests
------
+<a name="building"></a>
+### Building
+
+```sh
```
Comment (ded) [inline README.md:188]: `sh` ... you learn something new every day.

**Comment by ded** [comment]
ok. looks good to me. just re-reading some of these docs (just slightly reformatted) is reminding me how silly some these projects are organized...

anyhow, for a another time soon in the future. this looks ready to merge now, and iterate in the meantime.

File: README.md (hunk 2 above)
Comment (rvagg) [inline README.md:56]: I haven't played much but I think ids get stripped; and I've seen others use `name`, it's pretty safe and works; even works nicely on the new npm site: https://npmjs.org/package/traversty#first

Comment (ded) [inline README.md:56]: cool. good to know this too. that's nice.

**Pull request merged**